test_suite/tests/string.rs

```diff
@@ -15,6 +15,14 @@ mod sanitizers {
         assert_eq!(Name::new(" foo \n bar ").into_inner(), "foo \n bar");
     }
 
+    #[test]
+    fn test_trim_false() {
+        #[nutype(sanitize(trim = false))]
```

Comment:
Perhaps I just lack the context, but I don't understand what the difference is between `trim = false` and just not putting `trim` at all. From my fiddling around with the tests, it looks like there's none. 

I suppose you're just using it as a flag, and a sad consequence is that those tests are required. 
Just in case you haven't thought of that, `darling` [has a util for such flags](https://docs.rs/darling/0.20.1/darling/util/struct.Flag.html). Hope that helps ?

Reply:
@AlisCode 
> what the difference is between trim = false and just not putting trim at all.

There is no difference for the end user.

Both should be eventually turned into 
```rs
let sanitizers: Vec<StringSanitizer> = vec![]
```

However, the intermediate when parsing attributes are enums (not structs!) is different:
```
#[nutype(sanitize(trim = false))]
```
produces 
```rs
let sanitizers: Vec<RawStringValidator> = vec![RawStringValidator::Trim(false)]
```
while
```
#[nutype(sanitize())]
```
produces
```rs
let sanitizers: Vec<RawStringValidator> = vec![]
```

So the purpose of this test  to ensure that it's handled correctly in the match statement:
https://github.com/greyblake/nutype/pull/54/files#diff-73ce353ddefad1501374a253915ed97b273bb585ac8135c07b7519f44a2383a4R109-R110

Reply:
Thanks for `Flag`. I see, it's pretty much like `bool`, but more restrictive.
I've replaced `bool` with `Flag`: https://github.com/greyblake/nutype/pull/54/commits/903e9d9b09a9a10653f4ce7b77e1bb5181827724